### PR TITLE
Sbtopts

### PR DESCRIPTION
**Pull request by zeitos**
Travis was running the tests with some specific java opts configuration but most people forget to set them, so I'm moving them to a .sbtopts file

### CODE REVIEW

File: .sbtopts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+-J-Xmx1500m
```
Comment (velvia) [inline .sbtopts]: I would bump this to 2g, unless travis complains

**Comment by velvia** [comment]
Sweet thanks, feel free to merge after the build succeeds.

**Comment by zeitos** [comment]
interesting... seems like travis did not pick up the opts

**Comment by velvia** [comment]
Are you sure .sbtopts is picked up by Travis?  I thought Travis only cares about the .yml file

**Comment by zeitos** [comment]
sbtopts is picked by sbt and travis uses sbt... not 2.10 passed but not 2.11

**Comment by zeitos** [comment]
Interesting according to scalatest no test failed... 
ScalaTest
[info] Run completed in 3 minutes, 38 seconds.
[info] Total number of tests run: 1
[info] Suites: completed 1, aborted 0
[info] Tests: succeeded 1, failed 0, canceled 0, ignored 0, pending 0
[info] All tests passed.
[error] Error: Total 0, Failed 0, Errors 0, Passed 0

**Comment by zeitos** [comment]
@velvia according to http://www.scala-sbt.org/0.13/docs/Travis-CI-with-sbt.html travis should be using 
https://github.com/paulp/sbt-extras and the doc of sbt-extras says:
```
  -sbt-opts <path> file containing sbt args (if not given, .sbtopts in project root is used if present)
```

**Comment by zeitos** [comment]
Finally ... I got the results in 85d2391 .... it seems like it's loading both .jvmopts and .sbtopts 
# Executing command line:
/usr/lib/jvm/java-7-oracle/bin/java
-XX:MaxPermSize=512m
-Xms256m
-Xmx2g
-XX:+CMSClassUnloadingEnabled
-XX:+UseConcMarkSweepGC
-Dakka.test.timefactor=3
-XX:MaxPermSize=512m
-Xms256m
-Xmx2g
-XX:+CMSClassUnloadingEnabled
-XX:+UseConcMarkSweepGC
-Dakka.test.timefactor=3

**Comment by velvia** [comment]
@zeitos I restarted the 2.10 build.  Not clear to me why it failed.

**Comment by zeitos** [comment]
so... 7aa626e passed in 2.10 but not 2.11 and e90b47d passed on 2.11 but not 2.10... 
I'm still scratching my head

**Comment by velvia** [comment]
Hm, yeah I don't get it either.

**Comment by velvia** [comment]
I restarted the 2.11 build.  Very flaky.

**Comment by velvia** [comment]
@zeitos  Since the tests are now passing do we still need this?

**Comment by zeitos** [comment]
closing this one

**Pull request closed without merging**